xbmc/filesystem/bluray/BitReader.h

```diff
@@ -0,0 +1,172 @@
+/*
+ *  Copyright (C) 2005-2025 Team Kodi
```

Comment:
I think these shiny new files deserve a `Copyright (C) 2025 Team Kodi` header. It also sets the context in which the code was written.